src/mobile-v3/lib/src/app/dashboard/pages/location_selection/location_selection_screen.dart

```diff
@@ -63,7 +63,7 @@ class _LocationSelectionScreenState extends State<LocationSelectionScreen>
         currentState.response.measurements != null) {
       loggy.info('Dashboard already loaded, populating measurements');
```

Comment:
In the location selection screen, please remember to add the online_status query parameter while at it. You MUST only showcase ACTIVE locations and NOT all locations we have. That is something along the lines of `{{baseUrl}}/api/v2/devices/sites/summary?online_status=online`